test/minikube/minikube_rolling_upgrade_test.go

```diff
@@ -233,7 +233,7 @@ func TestKubernetesUpgradeFullDatabase(t *testing.T) {
 		verifyAllProcessesRunning(t, namespaceName, admin0, 2)
 	})
 
-	testlib.RunOnNuoDBVersionCondition(t, ">4.2.2", func(version *semver.Version) {
+	testlib.RunOnNuoDBVersionCondition(t, ">6.0.0", func(version *semver.Version) {
```

Comment:
Can you add a comment saying to keep this one protocol version greater than [`OLD_RELEASE`](https://github.com/nuodb/nuodb-helm-charts/blob/ccbb4afc627537d994d03b0e6a5b9b3bdee87970/test/minikube/minikube_rolling_upgrade_test.go#L27)?

The other option would be to build the version constraint so that it is `>N.0.0` where `N` is one greater than the major version appearing in `OLD_RELEASE`. I think just putting a comment here is fine.